charts/trino/README.md

```diff
@@ -175,65 +175,79 @@ Fast distributed SQL query engine for big data analytics that helps you explore
   ```
 * `resourceGroups` - object, default: `{}`  
 
-  Resource groups file is mounted to /etc/trino/resource-groups/resource-groups.json
-  Example:
+  [Resource groups control](https://trino.io/docs/current/admin/resource-groups.html)
+  Set the type property to either:
+  * `configmap`, and provide the Resource groups file contents in `resourceGroupsConfig`,
+  * `properties`, and provide configuration properties in `properties`.
+  Properties example:
+  ```yaml
+   type: properties
+   properties: |
+     resource-groups.configuration-manager=db
+     resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
```

Comment:
`localhost` is not the best example for apps deployed in Kubernetes, how about something similar to what we used in tests, like `trino-mysql.mysql.svc.cluster.local`?